list.md

```diff
@@ -7,175 +7,68 @@ librarians.
+| Germany     | Berlin          | [Staatsbibliothek zu Berlin](https://github.com/StaatsbibliothekBerlin)                                               |              |
+| Germany     | Berlin          | [University Library of Humboldt-Universität zu Berlin](https://github.com/UB-HU-Berlin)                               |              |
+| Germany     | Berlin          | [Universitätsbibliothek Technische Universität Berlin](https://github.com/tuub)                                       |              |
+| Germany     | Bochum          | [University Library Bochum](https://github.com/ubbochum)                                                              |              |
+| Germany     | Bonn            | [MPI Collective Goods Library Bonn](https://github.com/bibliocoll)                                                    |              |
+| Germany     | Bremen          | [Staats- und Universitätsbibliothek Bremen](https://github.com/suub)                                                  |              |
+| Germany     | Dresden         | [SLUB Saxon State and University Library Dresden](https://github.com/slub)                                            |              |
+| Germany     | Dortmund        | [Universitätsbibliothek Dortmund](https://github.com/UB-Dortmund)                                                     |              |
+| Germany     | Duisburg/Esssen | [Bibliothek der Universität Duisburg-Essen](https://github.com/unibib-duisburg-essen)                                 |              |
+| Germany     | Göttingen       | [Niedersächsische Staats- und Universitätsbibliothek Göttingen (SUB Göttingen)](https://github.com/subugoe)           |              |
+| Germany     | Göttingen       | [Gemeinsamer Bibliotheksverbund (GBV Göttingen)](https://github.com/gbv)                                              |              |
+| Germany     | Hamburg         | [Staats- und Universitätsbibliothek Hamburg](https://github.com/subhh)                                                |              |
+| Germany     | Hamburg         | [Universitätsbibliothek der TU Hamburg](https://github.com/tubhh)                                                     |              |
+| Germany     | Hamburg/Kiel    | [ZBW - Leibniz Information Centre for Economics](https://github.com/zbw)                                              |              |
+| Germany     | Hannover        | [Technische Informationsbibliothek (TIB), Hannover](https://github.com/TIBHannover)                                   |              |
+| Germany     | Hannover        | [Bibliothek der Hochschule Hannover 1](https://github.com/HsH-Bibliothek)                                             |              |
+| Germany     | Hannover        | [Bibliothek der Hochschule Hannover 2](https://github.com/hsh-bib)                                                    |              |
```

Comment:
I wouldn't make two entries for the HsH-Bib but rather give both links in the same row. The labels with 1 and 2 are not really meaningful.